mlir/utils/jenkins/Dockerfile.rocm

```diff
@@ -65,6 +65,9 @@ RUN ln -s /usr/bin/clang-10 /usr/bin/clang;\
     ln -s /usr/bin/git-clang-format-10 /usr/bin/git-clang-format;\
     ln -s /usr/bin/clang-format-diff-10 /usr/bin/clang-format-diff;\
     ln -s /usr/bin/lld-10 /usr/bin/lld
+RUN for binary in /usr/bin/llvm-*-10; do\
+    ln -s "$binary" "$(echo $binary | sed -e 's/-10//')";\
+    done
```

Comment:
What in addition than line 60 - 67 does it do? Why is it required for showing back-trace?

Reply:
This creates a bunch of links like `llvm-symbolizer -> llvm-symbolizer-10` (which is the one I wanted, but then I noticed there were a lot of other binaries like that running around). The backtrace generator seems to want an `llvm-symbolizer` in its path, and not an `llvm-symbolizer-10`

Reply:
I'd prefer to be consistent with the line 60-67, just so that we know explicitly which binary is called. So what you can do is to do is either add few more lines of `ln -s` to include the each binary you need; Or, be more specific than `llvm-*-10`, but `llvm-symbolizer-*-10`, for example.

Reply:
I insist that we should be more explicit/scoped about each `ln -s` in dockerfile. Just an action of being defensive, as this can have unintended effects if we are based on a different base image, for example.

Reply:
I see your point, this could just be a link to llvm-symbolizer, then.

(will fix when I'm not in the middle of some other things)